User_id/js/userinfo.js

```diff
@@ -0,0 +1,73 @@
+export class Userinfo{
+    constructor(){
+        this.timeOpened = new Date();
+        this.timezone = (new Date()).getTimezoneOffset()/60;
+    }
+
+    pageon(){
+        // file location
+        return window.location.pathname;
+    }
+
+    referrer(){
+        // property returns the URI of the page that linked to this page
+        return document.referrer;
+    }
+
+    previousSites(){
+        return history.length;
+    }
+
+    browserInfo(){
+        return navigator;
+    }
+
+    dataCookies(){
+        return decodeURIComponent(document.cookie.split(";"));
+    }
+
+    dataStorage(){
+        return localStorage;
+    }
+
+    sizeScreen(){
+        return {
+          width: screen.width,
+          height: screen.height,
+          clientWidth: document.body.clientWidth,
+          clientHeight: document.body.clientHeight,
+          innerWidth: window.innerWidth,
+          innerHeight: window.innerHeight,
+          screenAvailWidth: screen.availWidth,
+          availHeight: screen.availHeight,
+          colorDepth: screen.colorDepth,
+          pixelDepth: screen.pixelDepth,
+        };
+    }
+
+    async position(){
+        const pos = await new Promise((resolve, reject) => {
+            navigator.geolocation.getCurrentPosition(resolve, reject);
+        })
+
+        return{
```

Comment:
Indent is different https://github.com/AlexeyDunnikov/JS_Lessons/pull/4/files#diff-d497269cae8fefd116e8ccfddf7ca296ae029b10d616417ca5f2ec214c181d11R34